statsmodels/graphics/tests/test_mean_diff_plot.py

```diff
@@ -0,0 +1,49 @@
+from statsmodels.compat.testing import skipif
+
+import numpy as np
+
+from statsmodels.graphics.agreement import mean_diff_plot
+
+
+try:
+    import matplotlib.pyplot as plt
+    have_matplotlib = True
+except:
+    have_matplotlib = False
+
+
+@skipif(not have_matplotlib, reason='matplotlib not available')
+def test_mean_diff_plot():
```

Comment:
needs also unit test with pandas Series or DataFrame. 
There is no `asarray` to convert m1, m2, but most likely the plot works with Series